cmd/csaf_checker/processor.go

```diff
@@ -445,10 +445,10 @@ func (p *processor) integrity(
 		}
 		p.checkTLS(u)
 
-		// Check if the filename is confirming.
+		// Check if the filename is conforming.
 		p.badFilenames.use()
-		if !util.ConfirmingFileName(filepath.Base(u)) {
-			p.badFilenames.error("%s has not a confirming filename.", u)
+		if !util.ConformingFileName(filepath.Base(u)) {
+			p.badFilenames.error("%s has not a conforming filename.", u)
```

Comment:
A better phrasing would be "does not have a" or "has no" instead of "has not a".